src/ctia/entity/event/schemas.clj

```diff
@@ -15,10 +15,24 @@
    DeleteEventType])
 
 (s/defschema Update
-  {:field s/Keyword
-   :action s/Str
-   :change {:before s/Any
-            :after s/Any}})
+  (s/conditional
```

Comment:
I think that it breaks swagger UI.
If it does break it, you should opt for a more open schema
```clojure
{:field s/Keyword
 :action (s/enum "modified" "added" "deleted")
 :change (st/optional-keys
                  {:before s/Any
                    :after s/Any})}
```

Reply:
Thanks, I forgot about Swagger. Reverted to a simpler schema.

Reply:
it was an easy catch, I broke Swagger similarly few weeks ago :-)